src/backend/graphItemService.ts

```diff
@@ -31,7 +31,7 @@ export class GraphItemService {
    *    CREATE (b:BOOTSTRAP {result: ids}) RETURN b
    */
   static buildNodesQuery(category: string, keys: string[], values: unknown[]): string {
-    const node = `(n:\`${category}\` {` + keys.map((k, i) => `${k}: line[${i}]`).join(', ') + '})';
+    const node = `(n:\`${category}\` {` + keys.map((k, i) => `\`${k}\`: line[${i}]`).join(', ') + '})';
```

Comment:
Isn't there a good way to do escaping on all user input ?
Here I have a high feeling that we could do cypher injection